.github/workflows/ci_test-conda.yml

```diff
@@ -38,6 +38,18 @@ jobs:
         # Look to see if there is a cache hit for the corresponding requirements file
         key: pl-dataset
 
+    - name: Configure AWS credentials
+      uses: aws-actions/configure-aws-credentials@v1
+      with:
+        aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+        aws-secret-access-key: ${{ secrets.AWS_SECRET_KEY_ID }}
+        aws-region: us-east-1
+
+    - name: Copy files from S3
+      run: |
+        aws s3 cp --recursive s3://pl-public-data/legacy/checkpoints/ legacy/checkpoints/ #  --acl public-read
```

Comment:
Can we cache this ? Maybe use something like hash(`aws ls  s3://pl-public-data/legacy/checkpoints`)

Reply:
well, each model has 2KB so the total folder has about 80KB so we can add later if needed...
and then probably check your folder has all legacy checkpoint and if not pull...

Reply:
but adding todo... :]